File: Array/easy/pb_485/code.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int max_cnt = 0;
        for(int count=0,i=0;i<nums.size();i++){
            if(nums[i]==1) {
                count+=1;
                if(max_cnt<count) max_cnt=count;
                }
            else count=0;
        } 
        return max_cnt;
    }
};

File: Array/easy/pb_485/code.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count=0
        max_cnt=0
        for i in nums:
            count = count+1 if i==1 else 0
            if max_cnt<count: max_cnt=count
        return max_cnt